### DISCUSSION TITLE
Error on contract build

### DISCUSSION THREAD

**Discussion by paghost** [gochain/web3#235] [Q&A]
hello .. when i try to build :

web3 contract build PGHP.sol

i get this error 

ERROR: Failed to compile "PGHP_flatten.sol": solc: exec: "docker": executable file not found in $PATH

any help? thanks

**Comment by treeder** [comment]
You need to have Docker installed: https://docs.docker.com/get-docker/

**Comment by paghost** [comment]
it is already installed .version 1.5-2.

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by paghost** [comment]
i just moved my go to zeromint and it was easy to mint my nfts .. thanks for your help tho

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
